code401-challenges/src/test/java/code401/challenges/tree/BinarySearchTreeTest.java

```diff
@@ -0,0 +1,42 @@
+package code401.challenges.tree;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class BinarySearchTreeTest {
+    BinarySearchTree classUnderTest;
+    Node<Integer> root;
+    @Before
+    public void testInstantiateClass(){
+        classUnderTest = new BinarySearchTree();
+    }
+
+    @Test
+    public void testInstantiateEmptyTree(){
+        assertNull("This should return true if the tree is empty/null ", classUnderTest.getRoot());
+    }
+
+    @Test
+    public void testSingleRootNodeTree(){
+        classUnderTest.add(1);
+        assertTrue("This should return true if the tree contains a node ",classUnderTest.contains(1));
+    }
+
+    @Test
+    public void testLeftRightRootNodeTree(){
```

Comment:
This doesn't actually check if the node were correctly inserted into the left and right of the root node